docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
 
-. Finally configure the error handler for the connector. You can choose between *stop* (the connector shuts down in case of errors), *log* (the error is logged) or *dead letter queue* (the events that cannot be handled by the connector are sent to a dead letter topic of your Kafka instance).
-Select `log`.
+.. Click *Next*.
 
-. The next screen shows the configuration properties of your connector. Click *Create connector* to proceed with the deployment of the connector.
+.. Review the summary of the configuration properties of your Connectors instance and then click *Create Connectors instance* to deploy it.
++
+Your Connectors instance is listed in the table of Connectors. After a couple of seconds, the status of your Connectors instance changes to the *Ready* state and it starts producing messages and sending them to its associated Kafka topic.
 +
-After you complete the connector setup, the new connector is listed in the connectors table. After a couple of seconds, the connector moves to the *Ready* state. At this point the connector starts producing messages to the Kafka topic associated with the connector.
+From the connectors table, you can stop, start and delete your Connectors instance, as well as edit its configuration by clicking the options icon (three vertical dots).
 +
-From the connectors table, you can stop, start and delete the connector, as well as edit the connector configuration by clicking the options icon (three vertical dots).
+In the next procedure, you can verify that the source Connectors instance is sending messages as expected by creating a sink Connectors instance that consumes the messages.
 
 .Verification
 ifdef::qs[]
 * Have you completed these steps?
 endif::[]
 
 [id="proc-creating-sink-connector_{context}"]
-== Creating a {product-long-connectors} Sink Connector
+== Creating a Connectors instance for a data sink
 
 [role="_abstract"]
-In this step you create and configure a sink connector. A sink connector consumes events from an Kafka topic and sends them to an external system. In this quickstart you use the HTTP Sink connector which consumes the messages produced by the source connector configured in the previous step and calls a HTTP endpoint with the message payload.
+A sink connector consumes messages from a Kafka topic and sends them to an external system. In this quick start, you use the *HTTP Sink* connector which consumes the Kakfa messages (produced by the source Connectors instance) and sends the message payloads to an HTTP endpoint.
+
+ifndef::qs[]
+.Prerequisites
+* You're logged in to the {service-url-connectors}[^] web console.
+* You created the source Connectors instance as described in _Creating a Connectors instance for a data source_).
+* You have a unique URL from the https://webhook.site[webhook.site].
```

Comment:
Would users know what this is and how to get it? It's the first time it's mentioned in the quick start.

Reply:
it is a prerq in the YAML file. But I also need to add a conditional prereq here for the user guide - thanks!

Reply:
actually it is there - as a prereq in the user guide :)